packages/@okta/vuepress-site/docs/reference/saml-hook/index.md

```diff
@@ -368,6 +372,53 @@ When modifying an existing assertions statement, the path could begin with `/sub
 }
 ```
 
+## URI format response example
+
+This example displays `replace` and `add` operations with the URI formatted claim encoded.
+
+```JSON
+{
+  "commands": [
+    {
+      "type": "com.okta.assertion.patch",
+      "value": [
+        {
+          "op": "replace",
+          "path": "/claims/http:~1~1schemas.xmlsoap.org~1ws~12005~105~1identity~1claims~1foo/attributeValues/0/value",
+          "value": "replacementValue"
+        },
+        {
+          "op": "replace",
+          "path": "/claims/http:~1~1schemas.xmlsoap.org~1ws~12005~105~1identity~1claims~1foo/attributes",
+          "value": {
+            "attributes": {
+              "NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:basic"
+            }
+          }
+        },
+        {
+          "op": "add",
+          "path": "/claims/http:~1~1schemas.xmlsoap.org~1ws~12005~105~1identity~1claims~1foo",
```

Comment:
Can we change the claim name to `http:~1~1schemas.xmlsoap.org~1ws~12005~105~1identity~1claims~1bar`? Just realize that this is trying to `replace` and `add` the same claim name 🤦‍♀